libsolidity/codegen/Compiler.h

```diff
@@ -35,8 +36,9 @@ namespace solidity {
 class Compiler
 {
 public:
-	explicit Compiler(langutil::EVMVersion _evmVersion, OptimiserSettings _optimiserSettings):
+	explicit Compiler(langutil::EVMVersion _evmVersion, RevertStrings _revertStrings, OptimiserSettings _optimiserSettings):
```

Comment:
I know, you just added a parameter, but this `explicit` is superfluous. We'll never create a PR for cleaning up code - so my implicit question is, would it be "acceptable" to do such small changes when touching the code around anyways?

Reply:
Yep, that's what we should do!

Reply:
I am all for this, too. But I would prefer if we at least do that in extra commits?